frontend/src/components/profileCard/profileCard.scss

```
@@ -0,0 +1,177 @@
+@import '../../styles/index.scss';
+
+.wrapper {
+	width: 100%;
+	max-width: 265px;
+	margin-top: calc(-75px - #{$header-spacing}); // Subtract header-spacing due to padding on section wrapper
+	margin-right: $header-spacing;
+
+	@include mobile-all-portrait {
+		margin: 0 auto;
+	}
+}
+
+.profilePictureWrapper {
+	position: relative;
+	max-width: 150px;
+	margin: 0 auto;
+
+	.profilePicture {
+		position: relative;
+		display: flex;
+		justify-content: center;
+		width: 150px;
+		height: 150px;
+		border-radius: 100%;
+		background-size: cover;
+		background-position: center;
+		background-color: $darkest;
+		pointer-events: initial;
+
+		@include mobile-all-portrait {
+			overflow: visible;
+		}
+
+		@include box-shadow;
+
+		&Edit {
+			position: absolute;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			top: 0;
+			bottom: 0;
+			left: 0;
+			right: 0;
+			font-size: 2rem;
+			border-radius: 50%;
+			background-color: rgba(0,0,0,0.75);
+			color: white;
+			transition: 0.2s;
+			opacity: 0;
+	
+			&:hover {
+				transition: 0.2s;
+				opacity: 1;
+			}
+
+			@include mobile-all-portrait {
+				border-radius: 100%;
+			}
+	
+			input {
+				position: absolute;
+				width: 100%;
+				height: 100%;
+				opacity: 0;
+				cursor: pointer;
+			}
+		}
+	
+		&DraggingOver {
+			transition: 0.2s;
+			opacity: 1;
+			cursor: pointer;
+		}
+	}
+}
+
+.profileInfo {
+	margin-top: 10px;
+
+	.profileNameWrapper {
+		display: flex;
+		justify-content: center;
+		font-size: $nr-md-fontsize;
+		color: $primary;
+	}
```

```suggestion
	&NameWrapper {
		display: flex;
		justify-content: center;
		font-size: $nr-md-fontsize;
		color: $primary;
	}
```

Which means you change the name of this class to `.profileInfoNameWrapper`